### ISSUE TITLE
Vulnerabilites reported due to repo setup

### ISSUE THREAD

**Issue by olibooty** [Lumieducation/h5p-nodejs-library#2628]
Here is an [audit from socket.dev](https://socket.dev/npm/package/@lumieducation/h5p-react). It highlights a lack of source code repository reference in this particular package. I haven't checked the others yet, but I assume it may be the same?

**Comment by sr258** [comment]
@olibooty Yes, the other packages are the same. I don't see this as a real vulnerability, though, as there's no actual issue with the code or something that can be exploited. I'll leave the issue open and fix this in the next release of the packages. But I don't think it's urgent.
